bench-tps/src/cli.rs

```diff
@@ -27,6 +27,8 @@ pub enum ExternalClientType {
     // Submits transactions directly to leaders using a TpuClient, broadcasting to upcoming leaders
     // via TpuClient default configuration
     TpuClient,
+
```

Comment:
Would be nice to add comment here to detail where we send transactions (direct to leaders) and short summary of difference w/ respect to `TpuClient`